ext/date/tests/DatePeriod_wrong_recurrence_on_constructor.phpt

```diff
@@ -15,5 +15,5 @@ try {
 }
 ?>
 --EXPECT--
-DatePeriod::__construct(): The recurrence count '0' is invalid. Needs to be > 0
-DatePeriod::__construct(): The recurrence count '-1' is invalid. Needs to be > 0
+DatePeriod::__construct(): Argument #3 ($end) must be greater than 0
+DatePeriod::__construct(): Argument #3 ($end) must be greater than 0
```

Comment:
Due to the overloaded signature this error message is very awkward (in this case, the argument #3 is not the end date...) I think it would be best to avoid the argument error format here.

Reply:
Makes perfect sense, thanks for pointing this out

Reply:
This one is still unresolved. I'd go back to explicitly mentioning "recurrence count" rather than the argument number.